src/index.html

```diff
@@ -12,6 +12,142 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form
+      onsubmit="onSubmit()"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            required>
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name">
+        </div>
+        <div class="form-field">
+          <label for="old">How old are you?</label>
+          <input
+            type="number"
+            id="old"
+            name="old"
+            min="5"
+            max="130"
+            value="12" >
+        </div>
+        <div class="form-field">
+          <label for="birth">Full day of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth">
+        </div>
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            id="terms"
+            name="terms">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="email@example.com">
+        </div>
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="5"
+            maxlength="30">
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="loveCatsYes">Yes</label>
+          <input
+            type="radio"
+            id="loveCatsYes"
+            name="loveCats">
+          <label for="loveCatsNo">No</label>
+          <input
+            type="radio"
+            id="loveCatsNo"
+            name="loveCats">
+        </div>
+        <div class="form-field">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            id="color"
+            name="color">
+        </div>
+        <div class="form-field">
+          <label for="goBedTime">What time do you go to bed?</label>
+          <input
+            type="time"
+            id="goBedTime"
+            name="goBedTime">
+        </div>
+        <div class="form-field">
+          <label for="brendCars">What are your favorite brand of cars?</label>
+          <select
+            name="brendCars"
```

Comment:
Use correct English.
```suggestion
            name="brandCars"
```